public/vislib/vector_layer_types/EsLayer.js

```diff
@@ -157,8 +171,10 @@ export default class EsLayer {
 
       if (geo.type === 'point' || geo.type === 'geo_point') {
         layer.icon = `<i class="${options.icon}" style="color:${options.color};"></i>`;
+      } else if (geo.type.includes('line')) {
```

Comment:
please use same `===` logic as points. Just in case there is some other type, possibly in error, that includes `line`
```suggestion
      } else if (geo.type === 'linestring' || geo.type === 'multilinestring') {
```

Reply:
I think this should be just `=== 'line'`
since [DnD](https://github.com/sirensolutions/enhanced_tilemap/blob/6e0e24cf7c01d6015298ea85339449474559b39c/public/lib/dragAndDroplayercontrol/DndLayerControl.js#L592)

Reply:
This should be `=== line` I think.

Reply:
ok, so `.includes` can be removed?

Reply:
yes it's removed